checker/tests/ainfer-resourceleak/non-annotated/MustCallAliasOnRegularExits.java

```diff
@@ -0,0 +1,49 @@
+import java.io.IOException;
+import org.checkerframework.checker.calledmethods.qual.*;
+import org.checkerframework.checker.mustcall.qual.*;
+
+class MustCallAliasOnRegularExits {
+
+  @InheritableMustCall("a")
+  static class Foo {
+    void a() {}
+
+    int b() throws IOException {
+      return 0;
+    }
+  }
+
+  private class MCAConstructor extends Foo {
+
+    protected final @Owning Foo f; // expect owning annotation for this field
```

Comment:
Why do we have the comment if we are already writing `@Owning` in the test case?

Reply:
This comment added by mistake.